docs/concepts/training-data.mdx

```diff
@@ -5,19 +5,77 @@ sidebar_position: 4
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Data bundles
+# Training
 
-A _data bundle_ in Cradl is a model-specific collection of one or more [Datasets](datasets.mdx). Before training a model, you must specify a data bundle that the model will use as training data. Since the quality of the trained model depends on the quality of the training data, we require that any data bundles used for training must pass a statistical test in the form of a [Data Report](training-data.mdx#data-report) before training can begin.
+A _training_ in Cradl is the process of training a machine learning model on the data you provide.
+It can be summed up in 3 simple steps:
+1. User requests training in the Cradl app, or with one of our SDKs or CLI.
+2. Our team of Data Scientists verify that the training is feasible, and starts the training.
+3. User receives a new model that has been trained according to the request.
 
-## Creating a data bundle
+:::info
+We make sure to always provide a model whose architecture is based on the latest research to make sure that our customers get the most out of their data.
```

Comment:
Does it sound like we are not developing our own models?

Reply:
```suggestion
We develop models whose architecture is based on the latest research to make sure that our customers get the most out of their data.
```

Reply:
@staalezh Any comments?

Reply:
Is this relevant for understanding the training process? Maybe we can elaborate on this elsewhere, e.g. in a separate technology section?

Reply:
A valid point, I was thinking a little promo wouldn't harm, but if no one else agree, I am open to omit it.